test-one-file.lua

```diff
@@ -5,68 +5,218 @@
 -- LICENSE file in the root directory of this source tree. An additional grant
 -- of patent rights can be found in the PATENTS file in the same directory.
 
```

Comment:
You can add a line or two here on how to use it.
1. Follow the steps in - https://github.com/facebookresearch/wav2letter#running-the-decoder-inference
2. Example usage on librispeech pre-trained model - ~luajit ~/test-one-file.lua <PATH_TO_MODEL> <PATH_TO_WAV_FILE> -letters ~/librispeech-proc/letters-rep.lst  -words ~/dict.lst -lm ~/3-gram.pruned.3e-7.arpa -lmweight 3.1639 -beamsize 25000 -beamscore 40 -nthread 10 -smearing max